src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Automation/VsBuildManager.cs

```diff
@@ -29,13 +32,16 @@ internal VSBuildManager(IProjectThreadingService threadingService, IUnconfigured
         {
             AddEventSource(this);
             _threadingService = threadingService;
-
+            _unconfiguredProjectServices = unconfiguredProjectServices;
             Project = new OrderPrecedenceImportCollection<VSLangProj.VSProject>(projectCapabilityCheckProvider: unconfiguredProjectServices.Project);
         }
 
         [ImportMany(ExportContractNames.VsTypes.VSProject)]
         internal OrderPrecedenceImportCollection<VSLangProj.VSProject> Project { get; }
 
+        [Import]
+        internal Lazy<IDesignTimeInputsBuildManagerBridge, IAppliesToMetadataView>? DesignTimeInputsBuildManagerBridge { get; private set; }
+
```

Comment:
Why can't this be an import?

Reply:
It's a circular reference. Will add a comment in a follow up